### PR TITLE
totest manager fixes for Factory:ARM 

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: totest-manager.py [hunk 1]
```diff
@@ -707,7 +707,11 @@ def jobs_num(self):
 
 class ToTestFactoryARM(ToTestFactory):
     main_products = ['000product:openSUSE-cd-mini-aarch64',
-                     '000product:openSUSE-dvd5-dvd-aarch64']
+                     '000product:openSUSE-dvd5-dvd-aarch64',
+                     '000product:openSUSE-ftp-ftp-armv7hl',
+                     '000product:openSUSE-ftp-ftp-armv6hl',
+                     'JeOS'
```
Comment (DimStar77) [inline totest-manager.py]: Does that work? JeOS is not built in repo images (product_repo) arch local (product_arch)

```is_snapshottable``` is presumably never returning true in this case, as openSUSE:Factory:ARM/JeOS/images/local is never 'succeeded', but rather 'excluded'

Comment (dirkmueller) [reply totest-manager.py]: good point, this is correct. I'll postpone this to a followup commit, first I need to remodel it into a  :live subproject

**Comment by DimStar77** [comment]
Please adjust the commit message of e247655: it still mentions the addition of JeOS; other than that, :+1:

**Comment by dirkmueller** [comment]
@DimStar77 done

**Pull request merged**